### PR TITLE
changed from padding to margin

### PR DESCRIPTION
**Pull request by MathiasOhrstrom**
Now the navigationbar has the correct transparency

### CODE REVIEW

**Comment by hedvigoscar** [comment]
Pull Request build: https://app.bitrise.io/artifact/48419413/p/7653fce473aa20186d16ec264ac455df

**Comment by hedvigoscar** [comment]
Pull Request build: https://app.bitrise.io/artifact/48565166/p/5e343ff02e9b9bb7c1d3d5e5ada9d2a6

File: app/src/main/res/values-v29/styles.xml [hunk 1]
```diff
@@ -26,4 +26,10 @@
         <item name="colorControlNormal">@color/white</item>
     </style>
 
+    <style name="Hedvig.Theme.LoggedIn">
+        <item name="android:navigationBarColor">@android:color/transparent</item>
+        <item name="android:enforceNavigationBarContrast">false</item>
+        <item name="android:statusBarColor">@color/transparent</item>
```
Comment (hedvigoscar) [inline app/src/main/res/values-v29/styles.xml:32]:
```suggestion
        <item name="android:statusBarColor">@android:color/transparent</item>
```

File: app/src/main/res/values/styles.xml [hunk 2]
```diff
@@ -1,13 +1,13 @@
...
-    <style name="Hedvig.Theme" parent="Hedvig.Theme.V21" />
+    <style name="Hedvig.Theme" parent="Hedvig.Theme.V23" />
 
     <style name="Hedvig.Theme.KeyGear">
         <item name="android:navigationBarColor">@color/translucent_background</item>
     </style>
 
     <style name="Hedvig.Theme.LoggedIn">
-        <item name="android:navigationBarColor">@color/color_background_95</item>
+        <item name="android:navigationBarColor">#01FF0000</item>
```
Comment (hedvigoscar) [inline app/src/main/res/values/styles.xml:10]: What's this color?

File: app/src/main/res/values/styles.xml [hunk 3]
```diff
@@ -16,16 +16,20 @@
     </style>
 
     <style name="Hedvig.Theme.KeyGear.ItemDetail">
-        <item name="colorControlNormal">@color/white</item>
+        <item name="colorControlNormal">?colorOnPrimary</item>
```
Comment (hedvigoscar) [inline app/src/main/res/values/styles.xml:19]: Does this work with the toolbar in the Key Gear Item Detail-screen? I believe the choice of `white` was intentional here 😬

File: app/src/main/res/values/styles.xml [hunk 4]
```diff
@@ -16,16 +16,20 @@
...
         <item name="android:windowBackground">@color/hedvig_black</item>
+        <item name="android:statusBarColor">@color/hedvig_black</item>
+        <item name="android:windowLightStatusBar">false</item>
     </style>
 
-    <style name="Hedvig.Theme.Offer" parent="Hedvig.Theme.Offer.V21" />
+    <style name="Hedvig.Theme.Offer" parent="Hedvig.Theme.Offer.V23" />
 
     <style name="Hedvig.Theme.Splash">
+        <item name="android:windowLightStatusBar">false</item>
```
Comment (hedvigoscar) [inline app/src/main/res/values/styles.xml:32]: ❤️

**Review by hedvigoscar — CHANGES_REQUESTED** [review]

File: app/src/main/res/values/styles.xml (hunk 2 above)
Comment (MathiasOhrstrom) [reply app/src/main/res/values/styles.xml:10]: This is the almost transparent hack

File: app/src/main/res/values/styles.xml (hunk 3 above)
Comment (MathiasOhrstrom) [reply app/src/main/res/values/styles.xml:19]: I dont see any difference but i changed it back to white

Comment (hedvigoscar) [reply app/src/main/res/values/styles.xml]: `?colorOnPrimary` differs between light and dark theme while `@color/white` stays the same. Try it out in both themes to see the difference

File: app/src/main/res/values/styles.xml (hunk 2 above)
Comment (hedvigoscar) [reply app/src/main/res/values/styles.xml]: Is it required on this lower API level? Does the OS enforce contrast without it?

If yes, we should in the very least make sure it's not a very transparent red but probably a very transparent black

File: app/src/main/res/values/styles.xml (hunk 3 above)
Comment (MathiasOhrstrom) [reply app/src/main/res/values/styles.xml]: Yea i tried with white, black and pink. I didnt see any difference but I dont know what framework controls is so didnt know where to look

**Comment by hedvigoscar** [comment]
Pull Request build: https://app.bitrise.io/artifact/48682654/p/afba11457816601decd1d793f352fc56

File: app/src/main/res/values/styles.xml (hunk 3 above)
Comment (hedvigoscar) [reply app/src/main/res/values/styles.xml]: It's used for the toolbar actions, iirc

File: app/src/main/res/values/styles.xml (hunk 2 above)
Comment (MathiasOhrstrom) [reply app/src/main/res/values/styles.xml]: Okey, this was apparently not needed when tested on api 23, so it is now changed to `@android:color/transparent`

**Comment by hedvigoscar** [comment]
Pull Request build: https://app.bitrise.io/artifact/48685154/p/cc80d2f120aa51b4bcc236deed6eaad4

**Comment by hedvigoscar** [comment]
Bien hecho!

**Pull request merged**